web-share/resources/manual-helper.js

```diff
@@ -0,0 +1,71 @@
+// Sets up the page for running manual tests. Automatically creates the
+// instructions (based on the parameters) and the share button.
+function setupManualShareTest(
```

Comment:
I agree with @marcoscaceres here, I had to go back and forth multiple times to figure out what each one represented. Would make reviewing much easier.